test/cctest/test_node_postmortem_metadata.cc

```diff
@@ -140,16 +148,12 @@ TEST_F(DebugSymbolsTest, HandleWrapList) {
                                      .ToLocalChecked();
   TestHandleWrap obj(*env, object, &handle);
 
-  auto queue = reinterpret_cast<uintptr_t>((*env)->handle_wrap_queue());
-  auto head = queue +
-      nodedbg_offset_Environment_HandleWrapQueue__head___ListNode_HandleWrap;
-  auto next =
-      head + nodedbg_offset_ListNode_HandleWrap__next___uintptr_t;
-  next = *reinterpret_cast<uintptr_t*>(next);
+  auto last = tail + nodedbg_offset_ListNode_HandleWrap__next___uintptr_t;
+  last = *reinterpret_cast<uintptr_t*>(last);
```

Comment:
Isn't `last == head` after this? It looks wrong to me but we'll see what the CI says...

Reply:
The tail was retrieved before the new one was pushed back to the queue. So here the last would be expected to be the newly created one.

Reply:
That makes sense, thanks! Objection withdrawn then.